timed/employment/views.py

```diff
@@ -356,9 +367,18 @@ class PublicHolidayViewSet(ReadOnlyModelViewSet):
     def get_queryset(self):
         """Prefetch the related data.
 
+        Don't show public holidays to customers.
+
         :return: The public holidays
         :rtype:  QuerySet
         """
+        user = self.request.user
+
+        queryset = models.PublicHoliday.objects.all()
+
+        if CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Same here: don't "forbid to customers", but "allow to employees and admins" instead